docs/modules/concepts/pages/storage/index.adoc

```diff
@@ -1,4 +1,34 @@
-= Mailboxes and Email Storage
+= Storage
 :navtitle: Storage
 
-(TODO)
+== Overview
+
+Email is useful because any one person can send a message to any other person,
+anywhere in the world. The heart of email is the ability to transfer some message
+based on a  commonly-understood message format between two different systems that
+may have been developed entirely independently of each other. The concept is relatively
+simple, but the fact that email is so ubiquitious is simply a miracle.
+
+However, there is more to email than just the transfer of messages.
+
+Although it is possible to read a message, then discard it, never to be seen again,
```

Comment:
Email is by design asynchronous. People need to be able to send mail when I'm climbing, I need to review it afterward. 

In the meantime it needs to be stored somewhere.

Reply:
I agree, and that is kinda the point I am trying to make. Your example is a good one. I could add something like that. In any case I'll think about how to update this so it makes more sense.

Reply:
Updated text using this example.